manifests/sysmgmt.yaml

```diff
@@ -153,5 +153,5 @@ spec:
   # Spire service
   - name: spire
     source: csm-algol60
-    version: 1.6.0
+    version: 2.0.0
```

Comment:
The docker index file wasn't updated to use this new image: https://github.com/Cray-HPE/csm/blob/release/1.2/docker/index.yaml

Reply:
Since CASM-2670 was merged to 1.2 branch, we no longer use docker index file.